.env.test

```diff
@@ -0,0 +1,17 @@
+# Flask server configuration.
+FLASK_APP=todo_app/app
+FLASK_ENV=development
+FLASK_SKIP_DOTENV=True
+
+# Change the following values for local development.
+SECRET_KEY=secret-key
+
+# Trello API KEY AND Token conguration.
+APP_KEY=8b579065
+APP_TOKEN=67508832
+
+# Trello board,todo id conguration.
+BOARDID= '6032b49b5e077170a897e557'
```

Comment:
FYI this is not a Python file and should really be in this format:

BOARDID=6032b49b5e077170a897e557

Python-dotenv can read this but  you will have issues with Docker